addons/website_profile/views/website_profile.xml

```diff
@@ -67,105 +67,120 @@
+            <div class="mb-3 col-12">
+                <label class="text-primary mb-1 d-block" for="user_name"><span class="fw-bold">Name</span></label>
+                <div>
+                    <input type="text" class="form-control" name="name" id="user_name" required="True" t-attf-value="#{user.name}"/>
                 </div>
-            </form>
+            </div>
+            <div t-if="email_required" class="col-12">
+                <div class="alert alert-danger alert-dismissable oe_email_required p-2" role="alert">
+                    <button type="button" class="btn-close" data-bs-dismiss="alert"></button>
+                    Please enter a valid email address in order to receive notifications from answers or comments.
+                </div>
+            </div>
+            <div class="mb-3 col-6">
+                <label class="mb-1 text-primary" for="user_website"><span class="fw-bold">Website</span></label>
+                <div>
+                    <input type="text" class="form-control" name="website" id="user_website" t-attf-value="#{user.partner_id.website or ''}"/>
+                </div>
+            </div>
+            <div class="mb-3 col-6">
+                <label class="mb-1 text-primary" for="user_email"><span class="fw-bold">Email</span></label>
+                <div>
+                    <input type="text" class="form-control" name="email" id="user_email" required="True" t-attf-value="#{user.partner_id.email}"/>
+                </div>
+            </div>
+            <div class="mb-3 col-6">
+                <label class="mb-1 text-primary" for="user_city"><span class="fw-bold">City</span></label>
+                <div>
+                    <input type="text" class="form-control" name="city" id="user_city" t-attf-value="#{user.partner_id.city or ''}"/>
+                </div>
+            </div>
+            <div class="mb-3 col-6">
+                <label class="mb-1 text-primary"><span class="fw-bold">Country</span></label>
+                <div>
+                    <select class="form-select" name="country">
+                        <option value="">Country...</option>
+                        <t t-foreach="countries or []" t-as="country">
+                            <option t-att-value="country.id" t-att-selected="country.id == user.partner_id.country_id.id"><t t-esc="country.name"/></option>
+                        </t>
+                     </select>
+                </div>
+            </div>
+            <div class="mb-3 col-12">
+                <label class="mb-1 text-primary" for="description"><span class="fw-bold">Biography</span></label>
+                <textarea name="description" style="min-height: 120px"
+                    class="form-control o_wysiwyg_loader" placeholder="Write a few words about yourself..." t-att-content="user.partner_id.website_description"><t t-out="user.partner_id.website_description"/></textarea>
```

Comment:
"/image" does not work for portal users as they cannot access attachments.

I think there is a way to disable this command for portal maybe?
Note that current version doesn't support it either so if too complex / too annoying, we could ignore it.